zerver/views/documentation.py

```diff
@@ -187,17 +195,15 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
         if os.path.exists(article_absolute_path):
             with open(article_absolute_path) as article_file:
                 first_line = article_file.readlines()[0]
-            if context["article"] == "/zerver/api/api-doc-template.md":
+            if context["article"].endswith("api-doc-template.md"):
```

Comment:
I think an `if api_docs_view and ...` boolean here would be helpful for readability... and then code above can do things like `else: raise AssertionError("Invalid documentation view type")` rather than having the API docs just be an `else` clause above.

Reply:
This made sense to me, so each type of documentation view now has a boolean set to 'True'.